### PR TITLE
Mkchain readme updates

### PR DESCRIPTION
**Pull request by harryttd**
There were some other doc updates i failed to realize that should have been part of last PR. Whenever you guys get around to looking at this and approve of it,  feel free to squash merge yourselves if I'm not around.

### CODE REVIEW

File: mkchain/README.md [hunk 1]
```diff
@@ -53,15 +60,18 @@ You can explicitly specify some values by:
...
-| images.tezos                     | --docker-image     | Version of the Tezos docker image                                           | tezos/tezos:v8-release |
-| number_of_nodes                  | --number-of-nodes  | Number of non-baking nodes in the cluster                                   | 1                      |
-| number_of_bakers                 | --number-of-bakers | Number of baking nodes in the cluster                                       | 1                      |
-| rpc_auth                         | --rpc-auth         | Whether or not an [RPC auth](../rpc-auth/README.md) backend will be spun up | False                  |
-| zerotier_config.zerotier_network | --zerotier-network | Zerotier network id for external chain access                               |                        |
-| zerotier_config.zerotier_token   | --zerotier-token   | Zerotier token for external chain access                                    |                        |
+| YAML Key                         | mkchain argument         | Description                                                                 | Default                |
```
Comment (harryttd) [inline mkchain/README.md:63]: I chose not to include the gening of unsafe data flag here. It is meant primarily for people that need to test new networks and not for the common user. lmk if anyone disagrees.

**Review by nicolasochem — CHANGES_REQUESTED** [review]
The text all LGTM, but in spirit, I don't think we should brazenly encourage people to spin up a mainnet baker with the current state of the code.

Let's instead have 3 use cases:

* start a mainnet node (already done)
* start a florencenet node (with example values.yaml)
* start a florencenet baker (with example values.yaml)

And at the end of that, a big red warning that running a mainnet baker is not recommended at this stage, and that keys are better in a HSM that left online in a k8s secret.

**Comment by harryttd** [comment]
irl we discussed and decide to just remove the spin up baker section

**Comment by nicolasochem** [comment]
slack is not real life :)

**Comment by harryttd** [comment]
it's got to be more rl than gh tho

**Review by nicolasochem — APPROVED** [review]
thank you

**Pull request merged**